### DISCUSSION TITLE
Adding support of other languages

### DISCUSSION THREAD

**Discussion by LapsTimeOFF** [f1multiviewer/issue-tracker#158] [Feature requests]
It can be useful for people who struggle with English to add other languages like:
- French
- Italian
- Spanish
- Portuguese
- Dutch
- German

These are only examples, there can of course be more or less languages!

**Comment by ROoXZ1** [comment]
this is a great idea, I'm from Brasil, and know a lot of people that don't use F1MV because they don't  speak english, if you need any help translating to Portuguese when implementing this feature, i'm on the discord server (Anderson_#3310)

**Reply by LapsTimeOFF** [reply]
I'm here too for French

**Comment by f1multiviewer** [comment]
Just noticed this is actually a duplicate of https://github.com/f1multiviewer/issue-tracker/discussions/69

**Reply by LapsTimeOFF** [reply]
Sorry

**Reply by f1multiviewer** [reply]
No worries!
